patternfly/patternfly-elements
# feat: pfe-button | Add large variant #1745

## <Pull request description>

- Adding a new size variant property to pfe-button
- Options are "medium" and "large"
- Default size is "medium"
- Added css hook for controlling font-weight of the button


### Related issues

- (#1745 ) pfe-button | Add large variant #1745


### Preview


Link(s) to demo page(s) where this element can be viewed:
- [pfe-button demo](https://deploy-preview-1751--patternfly-elements.netlify.app/elements/pfe-button/demo/) 


### Testing instructions



- [ ] *Verify large variant size #1*
  1. Go here https://deploy-preview-1751--patternfly-elements.netlify.app/elements/pfe-button/demo/
  2. Using the dev tools, add an attribute of `<pfe-button size="large">`
  3. Verify that it changes size to match the large variant mockup.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
